### PR TITLE
Food code sweepups + heart boxes have more chocolate variety

### PR DESCRIPTION
**Pull request by dragomagol**
## About The Pull Request
Several things going on here:
- Chocolate boxes now contain bonbons (formerly tiny chocolates), truffles, and peanut butter cups
- Slices of bread now taste like their respective breads instead of tasting "indescribable"
- Fiesta skewers have food categories now
- Misc food file has had more split out of it into sweets.dm, vegetables.dm, and packaged.dm
- Removed a bunch of redundant food_reagent code
- Changed the sugar in clown cakes' food_reagents to banana juice
- Increased the amount of vitamin in pound cakes (it's four whole cakes and the amount of vitamin doesn't go up?)

## Why It's Good For The Game
Making code more readable for long lists of food reagents, smaller files to work with. Plus those new truffles were MADE to be put in that box. 

## Changelog
:cl: Tattle
add: chocolate boxes now contain bonbons (formerly tiny chocolates), truffles, and peanut butter cups
balance: changed the sugar in clown cakes' food_reagents to banana juice
balance: increased the amount of vitamin in pound cakes
fix: slices of bread now taste like their respective breads instead of tasting "indescribable"
fix: fiesta skewers are now MEAT and VEGETABLES
/:cl:

### CODE REVIEW

File: code/game/objects/items/food/_food.dm [hunk 1]
```diff
@@ -63,21 +63,22 @@
 	MakeGrillable()
 	MakeDecompose(mapload)
 	MakeBakeable()
-	make_microwavable()
+	MakeMicrowaveable()
```
[the remark this answers is not in the document]
Comment (JohnFulpWillard) [reply code/game/objects/items/food/_food.dm:66]: this is FUCKED

[the remark this answers is not in the document]
Comment (dragomagol) [reply code/game/objects/items/food/_food.dm:66]: Should they be all PascalCase or all snake_case? Having both is bleurgh

**Review by Twaticus — APPROVED** [review]

File: code/game/objects/items/food/_food.dm (hunk 1 above)
[the remark this answers is not in the document]
Comment (JohnFulpWillard) [reply code/game/objects/items/food/_food.dm:66]: It should be snake_case

File: code/game/objects/items/food/burgers.dm [hunk 2]
```diff
@@ -86,25 +113,33 @@
...
+		/datum/reagent/cyborg_mutation_nanomachines = 6,
+		/datum/reagent/consumable/nutriment/vitamin = 6,
+	)
 	tastes = list("bun" = 4, "lettuce" = 2, "sludge" = 1)
 	foodtypes = GRAIN | TOXIC
 	venue_value = FOOD_PRICE_EXOTIC
 
-/obj/item/food/burger/roburgerbig
-	name = "roburger"
+/obj/item/food/burger/roburger/big
```
Comment (JohnFulpWillard) [inline code/game/objects/items/food/burgers.dm:125]: Is there a ``/obj/item/food/burger/roburger``?

File: code/game/objects/items/food/frozen.dm [hunk 3]
```diff
@@ -74,7 +92,8 @@
 	icon_state = "flavorless_sc"
 	w_class = WEIGHT_CLASS_SMALL
 	trash_type = /obj/item/reagent_containers/cup/glass/sillycup //We dont eat paper cups
-	food_reagents = list(/datum/reagent/water = 11) // We dont get food for water/juices
+	food_reagents = list(/datum/reagent/water = 11,
+) // We dont get food for water/juices
```
Comment (JohnFulpWillard) [inline code/game/objects/items/food/frozen.dm:96]:
```suggestion
	food_reagents = list(
		/datum/reagent/water = 11,
	) // We dont get food for water/juices
```

File: code/game/objects/items/food/meatslab.dm [hunk 4]
```diff
@@ -371,6 +424,7 @@
 
 /obj/item/food/meat/slab/pig
 	name = "raw pork"
+	desc = "A slab of raw pork. This little piggy went to the butcher's."
```
Comment (JohnFulpWillard) [inline code/game/objects/items/food/meatslab.dm:427]: 😦

File: code/game/objects/items/food/packaged.dm [hunk 5]
```diff
@@ -0,0 +1,234 @@
+// Pre-packaged meals, canned, wrapped, and vended
+
+// Cans
+/obj/item/food/canned
```
Comment (JohnFulpWillard) [inline code/game/objects/items/food/packaged.dm:4]: Why is this a food wtf it's literally just canned drink with food reagents

File: code/game/objects/items/food/burgers.dm (hunk 2 above)
Comment (dragomagol) [reply code/game/objects/items/food/burgers.dm:125]: Yup! It should be the object just above the big roburger.

File: code/game/objects/items/food/packaged.dm (hunk 5 above)
Comment (dragomagol) [reply code/game/objects/items/food/packaged.dm:4]: Food code is very ancient in a lot of places 😔

**Pull request merged**